app/src/main/java/edu/grinnell/grinnell_publications_android/Utils.java

```diff
@@ -29,7 +31,26 @@
     static final String VERB_PAUSED = "paused";
     static final String VERB_RESUMED = "resumed";
 
-    private Utils(){
+    /** Author Contact Urls */
+    public enum AuthorContactUrl {
```

Comment:
Resolved per discussion with @Samee24. AuthorContactUrl may be used across the different layers. Keep as is for now.